src/clients/inst_kickoff.rb

```diff
@@ -34,21 +34,7 @@ def main
       # Feature #301903, bugzilla #244937
       if Mode.update
         # "/" means updating the running system, bugzilla #246389
-        if Installation.destdir == "/"
-          # When upgrading system, remove devs.rpm just from rpm database
-          LocalCommand(
-            "/bin/rpm -q 'devs' && /bin/rpm --nodeps --justdb -e 'devs'"
-          )
-          # normal upgrade
-        else
-          # When upgrading system, remove devs.rpm if installed
-          LocalCommand(
-            Builtins.sformat(
-              "/bin/rpm --root %1 -q 'devs' && /bin/rpm --nodeps --root %1 -e 'devs'",
-              Installation.destdir.shellescape
-            )
-          )
-
+        if Installation.destdir != "/"
```

Comment:
NP: Merge these two conditions? `if Mode.update && Installation.destdir != "/"`

Reply:
I am still not sure if we do not have more robust way to testing online upgrade. But fine for me to merge it.